android/assets/jsons/translations/German.properties

```diff
@@ -792,8 +792,7 @@ Requires = Benötigt
 Menu = Menü
 Brush Size = Pinselgröße
 Map saved = Karte gespeichert
- # Requires translation!
-Change map to fit selected ruleset? = 
+Change map to fit selected ruleset? = Karte ändern, um ausgewählten Regelsatz anzupassen?
```

Comment:
I would write it as "Karte an den ausgewählten Regelsatz anpassen?"
or when you really want to emphasise that the map gets changed: "Karte ändern, um sie an den ausgewählten Regelsatz anzupassen?"

Reply:
Yes, you're right. This was a mistake from my side. I think also your is better and more succinct.